core/src/components/input/input.ionic.scss

```diff
@@ -35,6 +35,18 @@
   min-height: 56px;
 }
 
+// Ionic Input Password Toggle Sizes
+// --------------------------------------------------
+
+:host ion-input-password-toggle {
+  --size: 40px;
+  --icon-size: #{globals.dynamic-font(16px)};
```

Comment:
```suggestion
```

Instead of setting these here, can we set them as the default values in `core/src/components/input-password-toggle/input-password-toggle.ionic.scss`?

Reply:
`--size` can must be set at the _**input.ionic.scss**_ otherwiese `--size` will not be override at the `ion-button` under the _**input-password-toggle.ionic.scss**_.
Already did that change.